kaniko/code/buildpackconfig/buildPackConfig.go

```diff
@@ -68,16 +68,7 @@ func NewBuildPackConfig() *BuildPackConfig {
 }
 
 func (b *BuildPackConfig) InitDefaults() {
-
-	logrus.Debug("Check if DOCKER_FILE_NAME env is defined...")
-	b.DockerFileName = util.GetValFromEnVar(DOCKER_FILE_NAME_ENV_NAME)
-	if b.DockerFileName == "" {
-		b.DockerFileName = defaultDockerFileName
-	}
-	logrus.Debugf("DockerfileName is: %s", b.DockerFileName)
```

Comment:
Removed as this information will be supplied from /layers/config/metadata.toml

Reply:
OK. So we have to review the README.md file to be sure that we do not support/use anymore a dockerfile but instead a TOML file containing the Dockerfile(s)

Reply:
(note that the code will still need to be updated further ...forthcoming... to reflect what's in the document, but I think this is closer to where we want to land WRT variable names, etc.)